src/tests/article.test.ts

```diff
@@ -0,0 +1,137 @@
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable @typescript-eslint/no-var-requires */
+/* eslint-disable dot-notation */
+import { ArticleModel } from '../entities/Article';
+import ArticleRepo from '../repos/ArticleRepo';
+import PublicationRepo from '../repos/PublicationRepo';
+import ArticleFactory from './data/ArticleFactory';
+import PublicationFactory from './data/PublicationFactory';
+
+import { dbConnection, disconnectDB } from './data/TestingDBConnection';
+
+// Maps an array of mongo documents [x] to an array of x.[val]
+function mapToValue(arr, val) {
+  return arr.map((x) => {
+    return x[val];
+  });
+}
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+function setAllToValue(arr, newMappings: { [key: string]: any }) {
+  return arr.map((x) => {
+    const newDoc = x;
+    Object.entries(newMappings).forEach(([k, v]) => {
+      newDoc[k] = v;
+    });
+    return newDoc;
+  });
+}
+
+function byDate(a, b) {
+  return -1 * (new Date(a.date).getTime() - new Date(b.date).getTime());
+}
+beforeAll(async () => {
+  await dbConnection();
+  await PublicationRepo.addPublicationsToDB();
+  await ArticleModel.createCollection();
+});
+
+beforeEach(async () => {
+  await ArticleModel.deleteMany({});
+});
+
+afterAll(async () => {
+  await ArticleModel.deleteMany({}).then(() => {
+    disconnectDB();
+  });
```

Comment:
I think we can do this here because rn we're just passing in a func which calls `disconnectDB` which is the same as just passing disconnectDB in directly:

```suggestion
  await ArticleModel.deleteMany({}).then(disconnectDB);
```